gui/marketBrowser.py

```diff
@@ -94,7 +110,7 @@ def toggleMetaButton(self, event):
             # Do 'nothing' if we're trying to turn last active button off
             if len(activeBtns) == 1 and activeBtns.pop() == ebtn:
                 # Keep button in the same state
-                ebtn.SetValue(True)

```

Comment:
@blitzmann I really didn't feel good about all the `SetValue` and `Enabled` calls scattered across this file. So I specialized the ToggleButton and made it pretty clear what we want to do, when we want to do it.

I tested this on Ubuntu and Windows and it works pretty well. macOS behaved like Ubuntu in the first round, so I don't anticipate any problems there.